features/lima/pkg.include

```diff
@@ -1,3 +1,11 @@
 cloud-init
 sshfs
 openssh-server
+dosfstools
+qemu-guest-agent
+efibootmgr
+efitools
+linux-image-$arch
+tpm2-tools
+ipmitool
```

Comment:
I don't think we need that, this seems to come from the metal feature and have no application in vms.